composer.json

```diff
@@ -41,7 +41,7 @@
     }
   ],
   "require": {
-    "basho/riak": "^3.3",
+    "basho/riak": "dev-features/lrb/gset-crdt-RIAK-2907",
```

Comment:
We have to tag `riak-php-client` first, then update this line, then we can merge this PR.